pom.xml

```diff
@@ -76,14 +76,14 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>4.3.1</version>
+      <version>5.4.0</version>
       <scope>compile</scope>
       <!--Dependency added by RoostGPT-->
     </dependency>
     <dependency>
-      <groupId>org.junit</groupId>
+      <groupId>junit</groupId>
```

Comment:
`Could not resolve dependencies for project com.medeiros:SPRINGProject:jar:0.0.1-SNAPSHOT: The following artifacts could not be resolved: org.junit:junit:jar:4.13.1 (absent): org.junit:junit:jar:4.13.1 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]`